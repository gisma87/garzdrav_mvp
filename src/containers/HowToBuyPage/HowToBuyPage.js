import React from "react";
import './HowToBuyPage.scss'

const HowToBuyPage = () => {
  return (
    <section className='HowToBuyPage wrapper'>
      <div className='HowToBuyPage__titleBlock'>
        <h1 className='HowToBuyPage__title'>Инструкция по оформлению заказа</h1>
      </div>

      <div className='HowToBuyPage__item'>
        <div className='HowToBuyPage__itemTitleBlock'>
          <div className='HowToBuyPage__number'>1</div>
          <h3 className='HowToBuyPage__textTitle'>Зарегистрируйтесь или авторизуйтесь на сайте</h3>
        </div>
        <p className='HowToBuyPage__itemDescription'>После регистрации вы сможете совершать заказы и отслеживать их
          статус в личном кабинете</p>
      </div>

      <div className='HowToBuyPage__item'>
        <div className='HowToBuyPage__itemTitleBlock'>
          <div className='HowToBuyPage__number'>2</div>
          <h3 className='HowToBuyPage__textTitle'>Найдите нужные лекарства в поиске</h3>
        </div>
        <p className='HowToBuyPage__itemDescription'>Воспользуйтесь поиском: укажите название лекарства, бренда или
          действующего вещества.
          Примените сортировку по цене или алфавиту, чтобы проще выбрать нужный товар.</p>
      </div>

      <div className='HowToBuyPage__item'>
        <div className='HowToBuyPage__itemTitleBlock'>
          <div className='HowToBuyPage__number'>3</div>
          <h3 className='HowToBuyPage__textTitle'>Добавьте товары в корзину</h3>
        </div>
        <p className='HowToBuyPage__itemDescription'>Нажмите на значок "Корзины" на карточке товара и добавьте нужное
          количество.
          Затем перейдите в корзину и нажмите "Выбрать аптеку"</p>
      </div>

      <div className='HowToBuyPage__item'>
        <div className='HowToBuyPage__itemTitleBlock'>
          <div className='HowToBuyPage__number'>4</div>
          <h3 className='HowToBuyPage__textTitle'>Выберите аптеку</h3>
        </div>
        <p className='HowToBuyPage__itemDescription'>Выберите из предложенного списка аптеку, где планируете забрать
          заказ.</p>
      </div>

      <div className='HowToBuyPage__item'>
        <div className='HowToBuyPage__itemTitleBlock'>
          <div className='HowToBuyPage__number'>5</div>
          <h3 className='HowToBuyPage__textTitle'>Проверьте контактную информацию</h3>
        </div>
        <p className='HowToBuyPage__itemDescription'>
          Проверьте номер телефона и email, на email придёт сообщение со статусом заказа, а на телефон может позвонить
          оператор, если возникнут вопросы при формировании заказа. Затем нажмите на "Оформить заказ"
        </p>
      </div>

      <div className='HowToBuyPage__item'>
        <div className='HowToBuyPage__itemTitleBlock'>
          <div className='HowToBuyPage__number'>6</div>
          <h3 className='HowToBuyPage__textTitle'>Готово! Заберите ваш заказ!</h3>
        </div>
        <p className='HowToBuyPage__itemDescription'>
          По Email мы сообщим, когда можно прийти в аптеку за заказом. В аптеке назовите номер телефона или номер
          заказа, оплатите и заберите заказ. Не забудьте взять с собой рецепт, если в заказе есть рецептурный препарат.
        </p>
      </div>
    </section>
  )
}

export default HowToBuyPage