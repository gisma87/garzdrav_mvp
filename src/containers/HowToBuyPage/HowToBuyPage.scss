@import "../../variables";

.HowToBuyPage {
  flex: 1 1;
  font-family: $primary-font;
  color: $primary-color;
  padding-bottom: 20px;

  &__titleBlock {
    margin: 10px 0 40px;
    font-size: 1.2rem;
    line-height: 1.5rem;

    @media (max-width: 900px) {
      line-height: 2.5rem;
    }
  }

  &__title {
    display: inline;
    font-weight: bold;
    margin: 0 0 20px;
    font-size: 1.6rem;
    line-height: 1.7rem;
    padding: 5px 0;
    border-bottom: 2px solid #1f5377;
  }

  &__item {
    padding-bottom: 30px;
  }

  &__itemTitleBlock {
    display: flex;
    align-items: center;
  }

  &__number {
    margin-right: 10px;
    height: 40px;
    min-width: 40px;
    background-color: $primary-color;
    border-radius: 50%;
    display: flex;
    justify-content: center;
    align-items: center;
    color: #fff;
    font-family: $secondary-font;
    font-size: 1.3rem;
    font-weight: bold;
  }

  &__textTitle {
    font-size: 1.4rem;
    @media (max-width: 900px) {
      font-size: 1.1rem;
    }
  }

  &__itemDescription {
    padding-left: 50px;
    font-size: 1.2rem;
    @media (max-width: 900px) {
      font-size: 1rem;
      margin-top: 10px;
    }
  }
}