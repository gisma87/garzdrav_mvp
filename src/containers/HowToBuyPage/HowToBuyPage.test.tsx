import React from "react";
import {render, screen} from "@testing-library/react";
import '@testing-library/jest-dom'
import HowToBuyPage from "./HowToBuyPage";

test('renders Инструкция', () => {
    render(<HowToBuyPage />);
    const linkElement = screen.getByText(/Инструкция/i);
    expect(linkElement).toBeInTheDocument();
});

test('snapshot HowToBuyPage', () => {
    const {asFragment} = render(<HowToBuyPage/>)
    expect(asFragment()).toMatchSnapshot()
})

