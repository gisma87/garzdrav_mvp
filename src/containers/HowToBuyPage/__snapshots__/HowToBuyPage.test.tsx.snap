// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot HowToBuyPage 1`] = `
<DocumentFragment>
  <section
    class="HowToBuyPage wrapper"
  >
    <div
      class="HowToBuyPage__titleBlock"
    >
      <h1
        class="HowToBuyPage__title"
      >
        Инструкция по оформлению заказа
      </h1>
    </div>
    <div
      class="HowToBuyPage__item"
    >
      <div
        class="HowToBuyPage__itemTitleBlock"
      >
        <div
          class="HowToBuyPage__number"
        >
          1
        </div>
        <h3
          class="HowToBuyPage__textTitle"
        >
          Зарегистрируйтесь или авторизуйтесь на сайте
        </h3>
      </div>
      <p
        class="HowToBuyPage__itemDescription"
      >
        После регистрации вы сможете совершать заказы и отслеживать их статус в личном кабинете
      </p>
    </div>
    <div
      class="HowToBuyPage__item"
    >
      <div
        class="HowToBuyPage__itemTitleBlock"
      >
        <div
          class="HowToBuyPage__number"
        >
          2
        </div>
        <h3
          class="HowToBuyPage__textTitle"
        >
          Найдите нужные лекарства в поиске
        </h3>
      </div>
      <p
        class="HowToBuyPage__itemDescription"
      >
        Воспользуйтесь поиском: укажите название лекарства, бренда или действующего вещества. Примените сортировку по цене или алфавиту, чтобы проще выбрать нужный товар.
      </p>
    </div>
    <div
      class="HowToBuyPage__item"
    >
      <div
        class="HowToBuyPage__itemTitleBlock"
      >
        <div
          class="HowToBuyPage__number"
        >
          3
        </div>
        <h3
          class="HowToBuyPage__textTitle"
        >
          Добавьте товары в корзину
        </h3>
      </div>
      <p
        class="HowToBuyPage__itemDescription"
      >
        Нажмите на значок "Корзины" на карточке товара и добавьте нужное количество. Затем перейдите в корзину и нажмите "Выбрать аптеку"
      </p>
    </div>
    <div
      class="HowToBuyPage__item"
    >
      <div
        class="HowToBuyPage__itemTitleBlock"
      >
        <div
          class="HowToBuyPage__number"
        >
          4
        </div>
        <h3
          class="HowToBuyPage__textTitle"
        >
          Выберите аптеку
        </h3>
      </div>
      <p
        class="HowToBuyPage__itemDescription"
      >
        Выберите из предложенного списка аптеку, где планируете забрать заказ.
      </p>
    </div>
    <div
      class="HowToBuyPage__item"
    >
      <div
        class="HowToBuyPage__itemTitleBlock"
      >
        <div
          class="HowToBuyPage__number"
        >
          5
        </div>
        <h3
          class="HowToBuyPage__textTitle"
        >
          Проверьте контактную информацию
        </h3>
      </div>
      <p
        class="HowToBuyPage__itemDescription"
      >
        Проверьте номер телефона и email, на email придёт сообщение со статусом заказа, а на телефон может позвонить оператор, если возникнут вопросы при формировании заказа. Затем нажмите на "Оформить заказ"
      </p>
    </div>
    <div
      class="HowToBuyPage__item"
    >
      <div
        class="HowToBuyPage__itemTitleBlock"
      >
        <div
          class="HowToBuyPage__number"
        >
          6
        </div>
        <h3
          class="HowToBuyPage__textTitle"
        >
          Готово! Заберите ваш заказ!
        </h3>
      </div>
      <p
        class="HowToBuyPage__itemDescription"
      >
        По Email мы сообщим, когда можно прийти в аптеку за заказом. В аптеке назовите номер телефона или номер заказа, оплатите и заберите заказ. Не забудьте взять с собой рецепт, если в заказе есть рецептурный препарат.
      </p>
    </div>
  </section>
</DocumentFragment>
`;
